Gemfile

```diff
@@ -25,7 +25,9 @@ gem 'prometheus_exporter'
 gem 'puma', '~> 5.1'
 gem 'rails', '~> 6.1.0'
 gem 'redis', '~> 4.2.5'
-gem 'sentry-raven'
+gem 'sentry-rails'
+gem 'sentry-ruby'
+gem 'sentry-sidekiq'
```

Comment:
I think these need to have their versions pinned, to be consistent with the rest of the `Gemfile`. Possibly:

```ruby
gem 'sentry-rails', '~> 4.1.5'
gem 'sentry-sidekiq', '~> 4.1.2'
```

Also (not a major issue) `sentry-ruby` is a dependency of `sentry-rails` and `sentry-sidekiq` so it doesn't need to be explicitly listed.

Reply:
👍 I have pinned the gemfile versions and removed sentry-ruby from the Gemfile.